### DISCUSSION TITLE
GIOTTO Flat screen support

### DISCUSSION THREAD

**Discussion by xfred81** [indilib/indi#1881] [Q&A] [answered]
Hello!

I'd like to purchase a real flat screen, and the GIOTTO models from Primaluce Lab seem interesting.
I see in git that they seem to be supported, but I also read a comment claiming this driver has not been tested.

What is the support status of GIOTTO flat screens in Indi?

Thanks for your reading!

**Comment by knro** [comment, marked as answer]
I'm almost done with the drivers for both ALTO+GIOTTO so they should be fully supported in INDI.

**Reply by xfred81** [reply]
Great news ;)

Yet, just to clarify, when you say "almost done", are they usable now or not ready for end-users?
For example, you say they "should be [....] supported" because you don't have/had much testers around?

Or are there others reasons to not show a bit more confidence maybe?

**Reply by knro** [reply]
Primaluce sent me the units, so they are guaranteed fully work. They should be released with the upcoming INDI release due in less than a week.

**Comment by xfred81** [comment]
Thanks a lot!

**Comment by xfred81** [comment]
I'm curious, is there any plan to add support for Primaluce's dedicated motorization too? (ALTO is the model name I think)

**Reply by knro** [reply]
Yes both will be supported.
